crates/adapters/src/format/json/input.rs

```diff
@@ -413,7 +466,7 @@ mod test {
 
     // Used to test RecordFormat::Raw.
     /*#[derive(Deserialize, PartialEq, Debug, Eq)]
-    struct OneColumn {
+        struct OneColumn {
```

Comment:
this is weird indentation. can this be deleted?

Reply:
Fixed.